packages/valory/services/impact_evaluator/service.yaml

```diff
@@ -132,6 +132,14 @@ extra:
         termination_from_block: ${TERMINATION_FROM_BLOCK:int:19717045}
         mech_chain_id: ${MECH_CHAIN_ID:str:gnosis}
         mech_interaction_sleep_time: ${MECH_INTERACTION_SLEEP_TIME:int:10}
+        use_mech_marketplace: ${USE_MECH_MARKETPLACE:bool:false}
+        mech_marketplace_config:
+          mech_marketplace_address: ${MECH_MARKETPLACE_ADDRESS:str:0x0000000000000000000000000000000000000000}
+          priority_mech_address: ${PRIORITY_MECH_ADDRESS:str:0x0000000000000000000000000000000000000000}
+          priority_mech_staking_instance_address: ${PRIORITY_MECH_STAKING_INSTANCE_ADDRESS:str:0x0000000000000000000000000000000000000000}
+          priority_mech_service_id: ${PRIORITY_MECH_SERVICE_ID:int:0}
+          requester_staking_instance_address: ${REQUESTER_STAKING_INSTANCE_ADDRESS:str:0x0000000000000000000000000000000000000000}
```

Comment:
Should we have some default addresses here?

Reply:
I would say yes, once we have them. For now we don't so let's keep it like this.